### PR TITLE
Add other tools page

### PR DESCRIPTION
**Pull request by dkillick**
Add a page that contains other tools (currently biggus, cf_units and iris_grib). The links for biggus and  iris_grib point to their respective readthedocs pages; cf_units points to the docs build on here.

Simultaneously addresses #133 and #134 :smile:

### CODE REVIEW

File: cartopy/documentation.html [hunk 1]
```diff
@@ -48,19 +48,34 @@
           "Getting started" and some step-by-step examples.</p>
           <h2>Latest</h2>
             <ul>
-                <li><a href="docs/latest/index.html">v0.14</a></li>
+                <li><a href="docs/latest/index.html">v0.13</a></li>
```
Comment (dkillick) [inline cartopy/documentation.html]: Oh good grief. I will fix this once again...

**Comment by bjlittle** [comment]
@dkillick Ping ... rebase please

**Comment by dkillick** [comment]
@bjlittle ping! Rebased.

**Comment by ajdawson** [comment]
There is no `tools.html` so the link in the toolbar is dead.

**Comment by ajdawson** [comment]
You perhaps need to run `build.py` to generate it?

**Comment by bjlittle** [comment]
@dkillick Give it a try with the rebuild ... and how did you render the test page? that we both eyeballed?

**Comment by ajdawson** [comment]
Perhaps just missed from the commit?

**Comment by dkillick** [comment]
@ajdawson @bjlittle it was just missed from the commit - the built html page is sat on my computer...

I'll add it in a follow-up PR.

**Comment by dkillick** [comment]
Thanks @ajdawson for bringing attention to the missing file - it's [all fixed now](http://scitools.org.uk/tools.html).

**Pull request merged**